### ISSUE TITLE
1.0.0-rc3 Missing function / classes for configuration

### ISSUE THREAD

**Issue by eyalcha** [kubernetes-client/javascript#1420]
The following doesn't exists 

```
const { createConfiguration, ServerConfiguration } = require('../dist');
const { PromiseMiddlewareWrapper } = require('../dist/gen/middleware');
```

https://github.com/kubernetes-client/javascript/blob/release-1.x/examples/patch-example.js

**Comment by SarthakNikhal** [comment]
Hey. Which functionality is broken because of this issue? Can I work on it?

**Comment by mstruebing** [comment]
@eyalcha I believe this should be available if we release another RC.
Can you validate if the issue is still the same on the `release-1.x` branch?

**Comment by mstruebing** [comment]
@eyalcha I've published a new release `1.0.0-rc4`: https://www.npmjs.com/package/@kubernetes/client-node/v/1.0.0-rc4
Could you confirm that this issue is solved in this release?
